server/schedule/checker/merge_checker.go

```diff
@@ -137,7 +137,7 @@ func (m *MergeChecker) Check(region *core.RegionInfo) []*operator.Operator {
 	log.Debug("try to merge region",
 		logutil.ZapRedactStringer("from", core.RegionToHexMeta(region.GetMeta())),
 		logutil.ZapRedactStringer("to", core.RegionToHexMeta(target.GetMeta())))
-	ops, err := operator.CreateMergeRegionOperator("merge-region", m.cluster, region, target, operator.OpMerge)
+	ops, err := operator.CreateMergeRegionOperator("merge-region", m.cluster, region, target, 0)
```

Comment:
```suggestion
	ops, err := operator.CreateMergeRegionOperator("merge-region", m.cluster, region, target)
```
I don‘t understand, Why kind will influence the operator priority? isn't it set by `SetPriorityLevel`?

Reply:
See https://github.com/tikv/pd/blob/f6fc8fb17af8c6495a282fa20d10e45422b9e02b/server/schedule/operator/operator.go#L61-L63.

Reply:
Why the kind is 0? I think it should be `OpMerge`? https://github.com/tikv/pd/blob/5fdb1fb78ba8e5ca0aff383d2c8652b2554e6bdd/server/schedule/operator/kind.go#L33-L35